pkg/ingester/series.go

```diff
@@ -16,7 +16,7 @@ var (
 	discardedSamples = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "cortex_ingester_out_of_order_samples_total",
```

Comment:
So do you think we should rename this metric then?

Reply:
yes

Reply:
done!